package.json

```diff
@@ -27,15 +44,17 @@
     "qrcode": "^1.2.0",
     "request": "^2.85.0",
     "request-promise-native": "^1.0.5",
-    "snyk": "^1.70.2",
     "web3": "^0.20.6",
     "y18n": "^4.0.0",
     "zxcvbn": "^4.4.2"
   },
-  "scripts": {
-    "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect",
-    "test": "snyk test"
+  "devDependencies": {
+    "nwjs-builder-phoenix": "^1.14.6"
   },
-  "snyk": true
+  "scripts": {
+    "build:app": "build --concurrent --tasks win-x86,winx64,linux-x86,linux-x64,mac-x64 --mirror https://dl.nwjs.io/ .",
+    "postinstall": "mkdir -p bin/gmc && cp node_modules/\\@musicoin/gmc/bin/gmc bin/gmc/ && mv node_modules/\\@musicoin/ux ./interface && yarn secondstage",
```

Comment:
I get this error log while trying to just `yarn`, so I'm assuming that this package wasn't included in the yarn dependencies or something similar.

```
cp: node_modules/@musicoin/gmc/bin/gmc: No such file or directory
error Command failed with exit code 1.
```

Reply:
Also, checkout the child repo for an issue I created just now.